nototools/generate_website_2_data.py

```diff
@@ -554,7 +554,7 @@ def get_css_generic_family(family):
 
 CSS_WEIGHT_MAPPING = {
     'Thin': 250,
```

Comment:
Sigh.  I should have looked in the font.  Xiangye said it should be 'ExtraLight 200' so that's what we did.  Only the name should have changed, the weight should have remained '250' according to this.